.github/workflows/snapshot.yml

```
@@ -93,6 +97,7 @@ jobs:
           # Build and push the snapshot images
           mvn -B -P container-image verify --no-transfer-progress -DskipTests \
             -Dquarkus.kubernetes.namespace='$${NAMESPACE}' \
+            -Dquarkus.container-image.labels."quay.expires-after"=${{ env.IMAGE_EXPIRATION }} \
```

I tried this locally to be sure, and the shell messes up the property unless it's surrounded by single quotes. After this, I am good if you merge. Thanks @hemahg 
```suggestion
            '-Dquarkus.container-image.labels."quay.expires-after"=${{ env.IMAGE_EXPIRATION }}' \
```